selection/select_ligand_and_surroundings/index.html

```diff
@@ -0,0 +1,14 @@
+<html>
+  <head>
+    <title>Parcel1 Sandbox</title>
```

Comment:
needs update

Reply:
Fixed in Paul's PR (which was merged here)